code/tests/core/test_identityState.brs

```diff
@@ -0,0 +1,127 @@
+' ********************** Copyright 2024 Adobe. All rights reserved. **********************
+
+' This file is licensed to you under the Apache License, Version 2.0 (the "License");
+' you may not use this file except in compliance with the License. You may obtain a copy
+' of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+' Unless required by applicable law or agreed to in writing, software distributed under
+' the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+' OF ANY KIND, either express or implied. See the License for the specific language
+' governing permissions and limitations under the License.
+
+' *****************************************************************************************
+
+' @BeforeEach
+sub TS_identityState_BeforeEach()
+    clearPersistedECID()
+end sub
+
+' @AfterEach
+sub TS_identityState_AfterEach()
+    clearPersistedECID()
+end sub
+
+' target: _init()
+' @Test
+sub TC_adb_IdentityState_init_noPersistedECID()
+    configurationModule = _adb_ConfigurationModule()
```

Comment:
For all these tests, why are you including the configuration module?  The IdentityState module doesn't appear to depend on it.

Reply:
I moved the tests from IdentityModule to IdentityState. I missed clearing it out, will remove them.